docs/README.md

```diff
@@ -7,7 +7,8 @@ description: Powerful CLI for managing Oasis networks, nodes, tokens and dApps
 
 Oasis command-line interface (CLI) is a powerful all-in-one tool for
 interacting with the Oasis Network. You can download the latest release
-binaries from the [GitHub repository].
+binaries from the [GitHub repository] (see the “Installing or Updating Oasis
+CLI on macOS” section below for macOS-specific instructions).
```

Comment:
Let's use ASCII in the markdown sources wherever possible. The HTML layer can later do more fancy rendering.

```suggestion
binaries from the [GitHub repository] (see the "Installing or Updating Oasis
CLI on macOS" section below for macOS-specific instructions).
```